static/app/components/charts/styles.tsx

```diff
@@ -13,8 +13,7 @@ export const SubHeading = styled('h3')`
 `;
 
 export const SectionHeading = styled('h4')`
-  display: inline-grid;
-  grid-auto-flow: column;
```

Comment:
Gonna test this change with visual diffs but this feels wrong to me. We should also move this component out of chart styles because wat